numpy/core/include/numpy/numpyconfig.h

```diff
@@ -19,6 +19,19 @@
         #define NPY_SIZEOF_LONG         4
         #define NPY_SIZEOF_PY_INTPTR_T  4
     #endif
+
+    #undef NPY_SIZEOF_LONGDOUBLE
+    #undef NPY_SIZEOF_COMPLEX_LONGDOUBLE
+
+    #ifdef __x86_64
+        #define NPY_SIZEOF_LONGDOUBLE         16
+        #define NPY_SIZEOF_COMPLEX_LONGDOUBLE 32
+    #elif defined(__arm64__)
+        #define NPY_SIZEOF_LONGDOUBLE         8
+        #define NPY_SIZEOF_COMPLEX_LONGDOUBLE 16
```

Comment:
I am trying to build with the macos-provided clang and getting weird errors because of this. @isuruf Shouldn't this be 16/32?

Reply:
>  @isuruf Shouldn't this be 16/32?

No, long double is equivalent to double on macos-arm64